WordPress/Classes/ViewRelated/Post/PostEditor+Publish.swift

```diff
@@ -517,7 +517,7 @@ extension PublishingEditor {
 
         PostCoordinator.shared.save(post)
 
-        dismissOrPopView(presentBloggingReminders: FeatureFlag.bloggingReminders.enabled)
+        dismissOrPopView(presentBloggingReminders: !JetpackNotificationMigrationService.shared.shouldDisableNotifications())
```

Comment:
We could add the feature flag here as well

Reply:
This is the issue of including `JetpackNotificationMigrationService` in the same targets that `FeatureFlag` is included. Extension targets don't support `UIApplication.shared`.

<img width="831" alt="image" src="https://user-images.githubusercontent.com/4062343/202741416-35973940-7bb5-421a-be7e-0345fc88ba30.png">